a_tour_of_textile.yml

```diff
@@ -0,0 +1,479 @@
+        code: '@todo'
+      android:
+        code: '@todo'
+    add_location_data_fail:
+      cmd:
+        code: "echo '{ \"latitude\": 91, \"longitude\": 2.294694 }' | textile files add --thread=\"12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw\"'"
+      js_http_client:
+        code: 'await Textile.files.add(JSON.stringify({ \"latitude\": 91, \"longitude\": 2.294694 }), "", "12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw")'
+      react_native:
+        code: '@todo'
+      ios:
+        code: '@todo'
+      android:
+        code: '@todo'
+    thread_leave:
+      cmd:
+        code: 'textile threads rm "12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw" --api="http://127.0.0.1:41600"'
+      js_http_client:
+        code: 'await textile2.threads.remove("12D3KooWBfdhD4tNMuTn5MHGof2bMZBKAUjFF3DBL3kuQQE5m1qw")'
+      react_native:
+        code: '@todo'
+      ios:
+        code: '@todo'
+      android:
+        code: '@todo'
+core:
+  setup:
+    js_http_client:
+      code: |
+            // no initialization needed, only import
+            import Textile from "@textile/js-http-client"
+    react_native:
+      code: '@todo'
+    ios:
+      code: '@todo'
+    android:
+      code: '@todo'
+  daemon:
+    cmd:
+      code: 'textile daemon --repo-dir="/tmp/buddy"'
+  logs:
+    cmd:
+      code: 'textile logs'
+  logs_text_only:
+    cmd:
+      code: 'textile logs --tex-only --level="info"'
+  init:
+    cmd:
+      code: 'textile init --seed="$(textile wallet init | tail -n1)" --repo-dir="/tmp/buddy" --swarm-ports="4101" --api-bind-addr="127.0.0.1:41600" --gateway-bind-addr="127.0.0.1:5150"'
+    js_http_client:
+      code: |
+            // You can override API settings to make use of the second peer you setup in CMD
+            import Textile as textile2 from "@textile/js-http-client"
+            // We'll use textile2 throught the rest of the tour to refer to this other peer
+            textile2.setOptions({port: 41600, url: 'http://127.0.0.1'});
+            // The new account address should match what the CMD returned on setup
+            await textile2.account.address() // P7X3gZus5H15tWCxk4oP6EVsgAM9vwUfCyepAKw49QuRyPYs
+    react_native:
+      code: '@todo'
+    ios:
+      code: '@todo'
+    android:
+      code: '@todo'
+  summary:
+    cmd:
+      code: 'textile summary'
+    js_http_client:
+      code: '@todo'
+    react_native:
+      code: '@todo'
+    ios:
+      code: '@todo'
+    android:
+      code: '@todo'
+  ping:
+    cmd:
+      code: 'textile ping 12D3KooWLh9Gd4C3knv4XqCyCuaNddfEoSLXgekVJzRyC5vsjv5d'
+    js_http_client:
+      code: '@todo'
+    react_native:
+      code: '@todo'
+    ios:
+      code: '@todo'
+    android:
+      code: '@todo'
+subscribe:
+  files:
+    cmd:
+      code: 'textile subscribe --type="files"'
+    js_http_client:
+      code: |
+            Textile.subscribe.stream("files")
+              .then((file) => {
+                // new file!
+              })
```

Comment:
not actually sure the above is right @carson? for how you parse that response.

also, i noticed the method docs were for all caps, "FILES" etc. but the CMD example here is using "files". Should we not care cap specificity in js?